src/services/image-api.ts

```diff
@@ -0,0 +1,30 @@
+import { fetchUserImage } from 'src/firebase/image-api';
+import { ImageOBJ } from 'src/types/profile-image';
+import { ConfigApi } from './config-api';
+
+export const ImageApi = ConfigApi.injectEndpoints({
+    endpoints: (build) => ({
+        getUserImage: build.query<string | null, ImageOBJ>({
+            /**
+             * Generating query for fetching and paginating users
```

Comment:
Is this image api needed, or can it be combined into the user api?

Reply:
This is more personal preference, I am not sure when a new RTK query API is needed, but I could combine them easily if you want.